src/viam/app/app_client.py

```diff
@@ -278,12 +280,58 @@ def proto(self) -> LogEntryPB:
         )
 
 
+class FragmentVisibility(str, Enum):
+    """
+    FragmentVisibility specifies who is permitted to view the fragment.
+    """
+
+    PRIVATE = "private"
+    """
+    Only visible to members in the fragment's organization.
+    """
+
+    PUBLIC = "public"
+    """
+    Visible to anyone and appears on the fragments page.
+    """
+
+    PUBLIC_UNLISTED = "public_unlisted"
+    """
+    Visible to anyone but does not appear on the fragments page.
+    """
+
+    UNSPECIFIED = "unspecified"
+    """
+    Uninitialized visibility.
+    """
+
+
 class Fragment:
     """A class that mirrors the `Fragment` proto message.
 
     Use this class to make the attributes of a `viam.proto.app.RobotPart` more accessible and easier to read/interpret.
     """
 
+    @classmethod
+    def visibility_from_proto(cls, visibility: FragmentVisibilityPB.ValueType) -> FragmentVisibility:
+        if visibility == FragmentVisibilityPB.FRAGMENT_VISIBILITY_PRIVATE:
+            return FragmentVisibility.PRIVATE
+        if visibility == FragmentVisibilityPB.FRAGMENT_VISIBILITY_PUBLIC:
+            return FragmentVisibility.PUBLIC
+        if visibility == FragmentVisibilityPB.FRAGMENT_VISIBILITY_PUBLIC_UNLISTED:
+            return FragmentVisibility.PUBLIC_UNLISTED
+        return FragmentVisibility.UNSPECIFIED
+
+    @classmethod
+    def visibility_to_proto(cls, visibility: FragmentVisibility) -> FragmentVisibilityPB.ValueType:
+        if visibility == FragmentVisibility.PRIVATE:
+            return FragmentVisibilityPB.FRAGMENT_VISIBILITY_PRIVATE
+        if visibility == FragmentVisibility.PUBLIC:
+            return FragmentVisibilityPB.FRAGMENT_VISIBILITY_PUBLIC
+        if visibility == FragmentVisibility.PUBLIC_UNLISTED:
+            return FragmentVisibilityPB.FRAGMENT_VISIBILITY_PUBLIC_UNLISTED
+        return FragmentVisibilityPB.FRAGMENT_VISIBILITY_UNSPECIFIED
```

Comment:
And move these methods to be inside of the enum class. Rename to simply `from_proto` and `to_proto`

Reply:
Also, `to_proto` should not be a class method in the new version